packages/ilp-plugin/CHANGELOG.md

```diff
@@ -7,75 +7,46 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 **Note:** Version bump only for package ilp-plugin
 
-
```

Comment:
Need a new version if it's a "breaking" change ?

Reply:
I think we'd version bump after this is merged with https://github.com/interledgerjs/interledgerjs#versioning

Reply:
When this is published I think it needs to be an `alpha` release initially, since ilp/pay is currently an alpha release.